src/main/java/com/direwolf20/buildinggadgets/common/utils/helpers/VectorHelper.java

```diff
@@ -31,4 +36,13 @@ public static BlockPos getPosLookingAt(EntityPlayer player) {
 
         return lookingAt.getBlockPos();
     }
+
+    //again, for more options
+    public static BlockPos getPosLookingAt(EntityPlayer player, RayTraceFluidMode fMode) {
```

Comment:
I know this is missing in the whole class, but could you maybe add an @Nullable to all Methdos in this class?